### PR TITLE
Lims/4350

### PR DESCRIPTION
**Pull request by omnia353**

### CODE REVIEW

File: api_testing/apis/test_plan_api.py [hunk 1]
```diff
@@ -320,11 +320,14 @@ def create_testplan_with_article_not_all(self):
         else:
             raise Exception(f'cant create the test plan with payload {payload}')
 
-    def create_completed_testplan_random_data(self):
-        random_article = random.choice(ArticleAPI().get_all_articles_json())
-        formatted_article = {'id': random_article['id'], 'text': random_article['name']}
-        material_type_id = GeneralUtilitiesAPI().get_material_id(random_article['materialType'])
-        formatted_material = {'id': material_type_id, 'text': random_article['materialType']}
+    def create_completed_testplan_random_data(self, formatted_article='', material_type=''):
```
Comment (ghada-37) [inline api_testing/apis/test_plan_api.py:323]: Why you updated this function while you can use function of: 
create_completed_testplan at line 228

**Review by ghada-37 — CHANGES_REQUESTED** [review]

**Comment by ghada-37** [comment]
@omnia353 You need to uncomment CI file

File: ui_testing/pages/order_page.py [hunk 2]
```diff
@@ -363,8 +364,8 @@ def get_suborder_table(self):
         self.info(' Get suborder table list.')
         self.base_selenium.click(element='order:suborder_list')
 
-    def create_new_suborder(self, material_type='', article_name='', test_plan='', test_unit='',
-                            add_new_suborder_btn='order:add_new_item'):
+    def create_new_suborder(self, material_type='', article_name='', test_plans=[''], test_unit='',
```
Comment (ghada-37) [inline ui_testing/pages/order_page.py:367]: You need to see all usages again as master updated now, msy be any merged test case use this function withvtest unit string not list

File: api_testing/apis/test_plan_api.py (hunk 1 above)
Comment (omnia353) [reply api_testing/apis/test_plan_api.py:323]: @ghada-37  create_complete_testplan wasn't working before and this function do the same as i want and i added the parameter as i want specific article

**Pull request closed without merging**